tests/foreman/api/test_contentview.py

```diff
@@ -1211,8 +1228,9 @@ def test_cv_dynflow_restart_promote(self):
         @status: Manual
         """
 
+    @tier2
     @stubbed()
-    def test_cv_dynflow_restart_publish(self):
+    def test_positive_dynflow_restart_publish(self):
```

Comment:
To have same approach as in UI and basically action should be located on third place
`test_positive_restart_publish_via_dynflow`